SIT.Manager/ViewModels/MainViewModel.cs

```diff
@@ -87,8 +87,15 @@ private void ManagerConfigService_ConfigChanged(object? sender, ManagerConfig e)
 
     private async Task CheckForUpdate()
     {
-        UpdateAvailable = await _appUpdaterService.CheckForUpdate();
-        SitUpdateAvailable = await _installerService.IsSitUpdateAvailable();
+        try
+        {
+            UpdateAvailable = await _appUpdaterService.CheckForUpdate();
+            SitUpdateAvailable = await _installerService.IsSitUpdateAvailable();
+        }
+        catch (Exception)
+        {
+
```

Comment:
Maybe put some logger here

Reply:
Oopsie, meant to add that just got distracted. Should be done now 😄